pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-7534-file-answer-one-to-many.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All this to add a  new auto generated primarykey/ID. Table should have been created with ID when it was created. 
May be deleting existing FK, then PK, adding new auto generated PK, filling it and later making it not null and PK can make not needing a temp table, deleting existing table , recreating table and updating from temp table.
I am ok with this temp table scenario too .

Reply:
It's a nicer solution, thanks